doc/api/cluster.md

```diff
@@ -649,6 +649,14 @@ finished.
 
 This can only be called from the master process.
 
+```js
+  const cluster = require('cluster');
+
+  if(cluster.IsMaster){
+    cluster.disconnect();
+  }
+```
```

Comment:
This example should ideally spawn a couple worker processes and at some point (maybe triggered by a worker) the main thread should call `.disconnect()`.